tests/testthat/test-multitables.R

```diff
@@ -190,6 +190,7 @@ with_mock_HTTP({
 
```

Comment:
Still want this; otherwise the code coverage report will show a drop and will prevent merging the pull request.